.github/workflows/ci.yml

```diff
@@ -7,6 +7,26 @@ on:
     branches: [ main ]
 
 jobs:
+  lint:
+    name: Lint C++
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install Clang Format
+        run: |
+          sudo apt-get update
+          sudo apt-get install clang-format
```

Comment:
clang-format gets stale fast for contemporary C++, we should make sure we're getting a current one.